### PR TITLE
zh-translation: /docs/ops/diagnostic-tools/controlz/index.md

### PR DESCRIPTION
**Pull request by arunfung**
ref: https://github.com/servicemesher/istio-official-translation/issues/1243

### CODE REVIEW

File: content/zh/docs/ops/diagnostic-tools/controlz/index.md [hunk 1]
```diff
@@ -1,34 +1,31 @@
...
 {{< /text >}}
 
-Here's sample of the ControlZ interface:
+这是 ControlZ 界面的示例：
 
 {{< image width="80%" link="./ctrlz.png" caption="ControlZ User Interface" >}}
 
-To access the ControlZ page of deployed components (i.e. Mixer, Galley, Pilot), you can port-forward their ControlZ endpoints
-locally and connect through your local browser:
+要访问已部署组件 (i.e. Mixer, Galley, Pilot) 的 ControlZ 页面，您可以用端口转发方式将 ControlZ 映射到本地，并通过本地浏览器进行访问：
```
Comment (rootsongjc) [inline content/zh/docs/ops/diagnostic-tools/controlz/index.md]: (i.e. Mixer, Galley, Pilot)  -> （例如 Mixer、Galley、Pilot）

Comment (arunfung) [reply content/zh/docs/ops/diagnostic-tools/controlz/index.md]: 已修改

**Pull request merged**